### PR TITLE
Support PHP 8 and Symfony 6

### PR DESCRIPTION
**Pull request by mwansinck**

### CODE REVIEW

**Comment by Eloar** [comment]
Would you like squash all those commits into one?

**Comment by mwansinck** [comment]
> Would you like squash all those commits into one?

Done

**Comment by Eloar** [comment]
@stephanvierkant @stwe @Seb33300 would you like to review and merge this code? It looks ok

File: composer.json [hunk 1]
```diff
@@ -18,24 +18,24 @@
...
+        "symfony/routing": "^4.4|^5.4|^6.0",
+        "symfony/security-core": "^4.4|^5.4|^6.0",
+        "symfony/translation": "^4.4|^5.4|^6.0",
         "twig/twig": "^2.9|^3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5|^8.5",
-        "friendsofphp/php-cs-fixer": "^2.15"
+        "phpunit/phpunit": "^8.5|^9.5",
+        "friendsofphp/php-cs-fixer": "^3.0"
```
Comment (Seb33300) [inline composer.json:38]: If you upgrade PHP-cs-fixer, you need to upgrade the config file `.php_cs.dist` to the new format `.php-cs-fixer.dist.php`

**Review by Seb33300 — CHANGES_REQUESTED** [review]

**Comment by mwansinck** [comment]
Do you also prefer that I commit al changes made by php-cs-fixer when running? 
It changes 13 files unrelated to my modifications but due to updated rules.

File: composer.json (hunk 1 above)
Comment (mwansinck) [reply composer.json:38]: Do you also prefer that I commit al changes made by php-cs-fixer when running?
It changes 13 files unrelated to my modifications but due to updated rules.

Comment (Seb33300) [reply composer.json:38]: IMO it should be better to upgrade php-cs-fixer in a separate PR first with all related changes
And then this PR only for PHP 8 / SF 6 upgrade

Comment (mwansinck) [reply composer.json:38]: Agreed that if possible that should be better.. however php-cs-fixer ^2 doesn't support Symfony 6 at all, so composer won't install Symfony 6 if I keep composer ^2 and I couldn't create this PR :-)

Comment (Seb33300) [reply composer.json:38]: Yes but I will merge the other PR before this one

Comment (mwansinck) [reply composer.json:38]: @Seb33300
https://github.com/stwe/DatatablesBundle/pull/984

**Comment by Seb33300** [comment]
PR for php-cs-fixer is merged.

Please rebase this PR

**Comment by mwansinck** [comment]
> PR for php-cs-fixer is merged.
> 
> Please rebase this PR

done :-)

**Review by stephanvierkant — APPROVED** [review]

**Review by Seb33300 — APPROVED** [review]

**Pull request merged**